bool navigation_status
bool go_to_speed_status
bool speed_status
